db/migrate/20200325185540_add_unique_index_to_profiles.rb

```diff
@@ -1,5 +1,9 @@
+require 'sidekiq/testing'
+
 class AddUniqueIndexToProfiles < ActiveRecord::Migration[5.2]
   def up
+    Sidekiq::Testing.inline!
```

Comment:
This causes sidekiq jobs to run synchronously, which is probably desired for all migrations. If they're run in redis and something fails, the migration will still pass, and the DB transaction will not get rolled back.

Reply:
How about calling `.perform` directly? It's what `:inline` does, without needing to require the testing files

Reply:
How do you propose doing that? We want this to be async most of the time, just not during this migration.

Reply:
The `destroy_policy_test_result` method can be overridden in the migration for the Profile class to call `perform`, unless there's some other job called I'm not aware of